guides/07_FAQs.md

```diff
@@ -106,6 +106,10 @@ Amazon Chime SDK limits are defined [here](https://docs.aws.amazon.com/chime/lat
 
 If your use case requires more than 250 attendees, consider using a [broadcasting solution](https://github.com/aws-samples/amazon-chime-meeting-broadcast-demo).
 
+### What happens to the subsequent participants who tries to turn on the local Video while 16 participants have already turned on the local video?
```

Comment:
- `local Video` => `local video`
- `participants who tries` => `participants who try`
- Use permalinks instead of line number links that can change in the future commits, e.g.,
  ```
   https://github.com/aws/amazon-chime-sdk-js/blob/bfc4c600fb7e68f2d358ecb6c7fd096d30b2d430/src/meetingsession/MeetingSessionStatusCode.ts#L73
   ```
- Remove "in the DefaultAudioVideoController" at the end. See the below bullet point.
- `videoSendDidBecomeUnavailable` => I don't think we need to link the implementation detail. Instead, you can just add a link to the AudioVideoObserver API reference.
  https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideoobserver.html#videosenddidbecomeunavailable
- End a sentence with a period.